docker-compose.yml

```diff
@@ -25,10 +25,10 @@ services:
     logging: *default-logging
     environment:
       - SIA_MODULES=gctwra
-      - SKYD_DISK_CACHE_ENABLED=false
-      - SKYD_DISK_CACHE_SIZE=53690000000 # 50GB
-      - SKYD_DISK_CACHE_MIN_HITS=3
-      - SKYD_DISK_CACHE_HIT_PERIOD=3600 # 1h
+      - SKYD_DISK_CACHE_ENABLED=${SKYD_DISK_CACHE_ENABLED:-false}
+      - SKYD_DISK_CACHE_SIZE=${SKYD_DISK_CACHE_SIZE:-53690000000} # 50GB
```

Comment:
it's 50GiB 😛

Reply:
![Screenshot 2022-01-12 at 13 59 06](https://user-images.githubusercontent.com/3755029/149144830-362de33b-e612-4443-88ab-9bf41bd0d9ea.png)

random library from npm cannot be wrong, sorry